src/components/AssetClassification/FeeDestination.tsx

```diff
@@ -5,10 +5,17 @@ import { RemoveButton } from "../Button";
 import { InputOrDisplay } from "../Input";
 import { EntityDetailDisplay } from "./EntityDetailDisplay";
 
-const FeeDestinationWrapper = styled.div`
+const FeeDestinationControlWrapper = styled.div<{ isNotLastElement: boolean }>`
+    display: flex;
+    gap: 10px;
+    padding-bottom: ${({ isNotLastElement }) => isNotLastElement && '25px' };
```

Comment:
this works, but you could probably just target this with pure css like 
```suggestion
   &:not(:last-child) {
         padding-bottom: 25px;
    }
```

and then not have to calculate/pass that prop in

Reply:
:dank: That's a way more doper way to handle this, thanks.  I'll pop that tweak in there 🙏